motionplan/stubs_nocgo.go

```diff
@@ -0,0 +1,50 @@
+//go:build no_cgo
+
+package motionplan
+
+import (
+	"math/rand"
+
+	"github.com/pkg/errors"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+)
+
+const pathdebug = false
+
+var errNotSupported = errors.New("not supported on this build")
+var flipPose = spatialmath.NewPoseFromOrientation(&spatialmath.OrientationVectorDegrees{OZ: 1, Theta: 180})
```

Comment:
what is this?

Reply:
`pathdebug` and `flipPose` are globals that are defined in motionplan/tpSpaceRRT.go -- it's a cgo file, had to redefine them here. could move to a common location I guess

Reply:
Thats ok you don't need to fix our tech debt as a result of this PR.